pandas/tests/base/conftest.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
```

Comment:
Here's a link to the original mixin this aims to replace:
https://github.com/pandas-dev/pandas/blob/master/pandas/tests/base/test_ops.py#L43-L83